index.html

```diff
@@ -29,8 +29,8 @@ <h1 id="members-count">217</h1>
         issues
       </div>
       <div class="card-body">
-        <h1 class="github-stat"><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Avoxpupuli+archived%3Afalse+sort%3Acreated-asc">oldest</a></h1>
-        <h1 class="github-stat"><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Avoxpupuli+archived%3Afalse+sort%3Acreated-desc">newest</a></h1>
+        <h1 class="github-stat"><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Avoxpupuli+archived%3Afalse+sort%3Acreated-asc">oldest</a></h1>
+        <h1 class="github-stat"><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Avoxpupuli+archived%3Afalse+sort%3Acreated-desc">newest</a></h1>
```

Comment:
Looks like changing "issues" with "search" seems to do the right thing.
```suggestion
        <h1 class="github-stat"><a href="https://github.com/search?q=is%3Aopen+is%3Aissue+org%3Avoxpupuli+archived%3Afalse+sort%3Acreated-asc">oldest</a></h1>
        <h1 class="github-stat"><a href="https://github.com/seach?q=is%3Aopen+is%3Aissue+org%3Avoxpupuli+archived%3Afalse+sort%3Acreated-desc">newest</a></h1>
```
Same thing a few lines bellow with "pulls": changing it to "search" fix the search.